docs/PROJECTS.csv

```diff
@@ -120,4 +120,5 @@ VS Code Kubernetes Tools,N/A,N/A,11/09/2021,Sandbox,CLA,No,Yes,No,Yes,N/A?,https
 Nocalhost,N/A,N/A,11/16/2021,Sandbox,No,No,No,No,Yes,https://nocalhost.slack.com/,,
 KubeArmor,N/A,N/A,11/16/2021,Sandbox,No,No,No,No,Yes,https://kubearmor.herokuapp.com/,https://github.com/kubearmor/KubeArmor,
 K8up,N/A,N/A,11/16/2021,Sandbox,No,No,No,No,Yes,N/A?,https://github.com/k8up-io/k8up,
-kube-rs,N/A,N/A,11/16/2021,Sandbox,DCO,No,Yes,No,Yes,Discord,https://github.com/kube-rs/kube-rs,
\ No newline at end of file
+kube-rs,N/A,N/A,11/16/2021,Sandbox,DCO,No,Yes,No,Yes,Discord,https://github.com/kube-rs/kube-rs,
+Devfile,N/A,N/A,1/11/2022,Sandbox,No,No,No,No,Yes,N/A?,https://github.com/devfile/api,

```

Comment:
Hi @amye, I am the submitter of the Devfile proposal. Thank you for this PR. I would like to suggest 2 minor changes:
- we don't use Slack but we do use Gitter for community IM (https://gitter.im/devfile/community)
- replace the GitHub repository with the Devfile GitHub organization https://github.com/devfile/ as I see that's what other projects have  

```suggestion
Devfile,N/A,N/A,1/11/2022,Sandbox,No,No,No,No,Yes,https://gitter.im/devfile/community,https://github.com/devfile/,
```